### PR TITLE
[CI/CD] Cleanup use of install command to use composer ezplatform-install everywhere

### PR DESCRIPTION
**Pull request by andrerom**
Changes all scripts (Docker, Platform.sh, Travis) to use `composer ezplatform-install` consistently to get rid of customizations per repo to specify install argument several places.

Needed to avoid further mess in https://github.com/ezsystems/ezcommerce-advanced/pull/6

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -81,8 +81,7 @@ hooks:
         set -e
         # Mainly relevant for eZ Platform demo usage, for own setup adapt this or remove and rely on migrations.
         if [ ! -f web/var/.platform.installed ]; then
-            . ./.env
-            php -d memory_limit=-1 bin/console ezplatform:install $INSTALL_EZ_INSTALL_TYPE
+            php -d memory_limit=-1 `which composer` ezplatform-install
```
Comment (andrerom) [inline .platform.app.yaml]: @vidarl just to check, do we really need to set `-d memory_limit=-1` when running installer?

Comment (andrerom) [reply .platform.app.yaml]: Edit, I see [blame of this](https://github.com/ezsystems/ezplatform/pull/169) is @glye

Comment (glye) [reply .platform.app.yaml]: Hm, I don't remember anymore if that was necessary or just adhering to tradition :)
Anyway, now we can set the php memory limit separately here in the same app file: https://docs.platform.sh/languages/php/ini.html
But uncertain if we can have separate settings for cli and sapi.

File: bin/.travis/prepare_ezpublish.sh [hunk 2]
```diff
@@ -28,4 +28,4 @@ echo "> Run assetic dump for behat env"
 php app/console --env=behat --no-debug assetic:dump
 
 echo "> Installing ezplatform clean"
-php app/console --env=behat ezplatform:install clean
```
Comment (mnocon) [inline bin/.travis/prepare_ezpublish.sh]: Given that it still used app/console, meaning that it's not meant for v2 - can we remove this file?
If it is used, then the new approach will fail, as `ezplatform-install` was introduced in 2.2 or something like that (and won't be available anyway).

File: .platform.app.yaml (hunk 1 above)
Comment (andrerom) [reply .platform.app.yaml]: Ok, seeing they have default of 128mb and the fact that this command will potentially triggering kaliop migrations on install..

... keeping this 😅 
Thanks for the input  @glye

File: bin/.travis/prepare_ezpublish.sh (hunk 2 above)
Comment (andrerom) [reply bin/.travis/prepare_ezpublish.sh]: yes we can, most things in this folder was for travis `dist: precise`, so oldie stuff.

Comment (andrerom) [reply bin/.travis/prepare_ezpublish.sh]: done in 30dc6fc6c930204f9b3f531491e4182c2417ab53

**Pull request merged**